### PR TITLE
address_type is null after setting billing and shipping addresses on cart

### PR DESCRIPTION
**Pull request by XxXgeoXxX**
### Description (*)
Issue: https://github.com/magento/graphql-ce/issues/409

#### Preconditions (*)
1. Customer is created
2. Simple Product is created

#### Steps to reproduce (*)
1. Generate Customer token (GraphQL)
2. Create empty cart (GraphQL)
3. Add Simple Product to Cart (GraphQL)
4. Set shipping address on cart (GraphQL)
5. Set billing address on cart (GraphQL)
6. Perform query

```
query {
  cart(
    cart_id: "{{ CART_ID }}"
  ) {
    available_payment_methods {
      title
      code
    }
    billing_address {
      address_type
      available_shipping_methods {
        carrier_code
        method_code
      }
    }
    shipping_addresses {
      address_type
      available_shipping_methods {
        carrier_code
        method_code
        amount
      }
    }
  }
}
```

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/QuoteGraphQl/etc/schema.graphqls [hunk 1]
```diff
@@ -211,11 +211,6 @@ type SelectedPaymentMethod {
 type SelectedPaymentMethodAdditionalData {
 }
 
-enum AdressTypeEnum {
```
Comment (naydav) [inline app/code/Magento/QuoteGraphQl/etc/schema.graphqls]: Please, revert this change
it is better to use the `ENUM` if you know in advance the list of possible options
Thanks

File: app/code/Magento/QuoteGraphQl/Model/Cart/ExtractDataFromAddress.php [hunk 2]
```diff
@@ -42,6 +42,7 @@ public function execute(QuoteAddress $address): array
 
         $addressData = array_merge($addressData, [
             'address_id' => $address->getId(),
+            'address_type' => $address->getAddressType(),
```
Comment (naydav) [inline app/code/Magento/QuoteGraphQl/Model/Cart/ExtractDataFromAddress.php:45]: Please, add check to API-functional tests
```
\Magento\GraphQl\Quote\SetShippingAddressOnCartTest
\Magento\GraphQl\Quote\Customer\SetBillingAddressOnCartTest
\Magento\GraphQl\Quote\Guest\SetBillingAddressOnCartTest

**Review by naydav — CHANGES_REQUESTED** [review]

File: app/code/Magento/QuoteGraphQl/etc/schema.graphqls (hunk 1 above)
Comment (XxXgeoXxX) [reply app/code/Magento/QuoteGraphQl/etc/schema.graphqls]: Changes were reverted

File: app/code/Magento/QuoteGraphQl/Model/Cart/ExtractDataFromAddress.php (hunk 2 above)
Comment (XxXgeoXxX) [reply app/code/Magento/QuoteGraphQl/Model/Cart/ExtractDataFromAddress.php:45]: Tests were updated
